### ISSUE TITLE
parsing of <htmlnote> from old maps not working with QXmlStreamReader yet

### ISSUE THREAD

**Issue by atuctuc** [insilmaril/vym#64]
i get an error, when i open some old vym files.

e.g.
Citical Parse error
Invalid XML character,
Lne 1037, column 260

the maps are not correctly displayed after loading

**Comment by insilmaril** [comment]
Which version of vym are you using?

Can you attache the old map? If not, could you unzip it and send me the 

- the beginning of the file, there is a vymVersion="x.y.z" and
- the offending line

Thx

**Comment by atuctuc** [comment]
```
all files were used/created with  : Version 2.7.510 - 2020-08-21

the problems with some files apears with: Version 2.9.510 i compiles today on linux

<?xml version="1.0" encoding="utf-8"?><!DOCTYPE vymmap>
<vymmap  version="1.11.3" author="" comment="" date="2007-12-19" backgroundColor="#ffff7f" selectionColor="#ffff00" linkStyle="StylePolyParabel" linkColor="#0000ff" defXLinkColor="#e6e6e6" defXLinkWidth="1">

the error is: line 6, column 36
        <htmlnote  fonthint="undef">





```

**Comment by atuctuc** [comment]
version="1.11.3" author="" comment="" date="2007-12-19"

      <htmlnote  fonthint="undef">

**Comment by insilmaril** [comment]
Ah, I already suspected it: 2.9.510 is a rewrite in huge parts and XML parsing is not complete yet. Exactly the <vymnote> tag is going to be added in the next 2-3 weeks, then some testing with such old maps would be very helpful, actually I'd like to add some to the test suite.

2.9.510 is not meant for production use yet, please use 2.9.0 (from release branch) for real work. There are more things which are not completely ported yet... I hope to get it stable enough in the next 8 weeks so that I can start using it in daily work, but it will take longer before I'd call it "beta".

<vymnote> should be read by 2.9.0, which will again save it in a format readable by 2.9.510

**Comment by atuctuc** [comment]
ok, i compiled now version 2.9. 

it seems to worke with my old maps!

it think there is also a bug in the notes editor. 
i can set the fontcolor, but the background color is always black. 
the menu entries in the notes menue are not taken over

**Comment by insilmaril** [comment]
Ah thanks, I opened #65  for that.

**Comment by insilmaril** [comment]
This has been fixed in 2.9.511.

While backporting the tests to 2.9.0 I found that even 2.9.0 seems to have problems with legacy maps: <htmlnote> was parsed, but no notes generated!

2.9.1 should work correctly.

@atuctuc - can you test your maps with 2.9.1?  (release branch)    development branch should work, too now.

I also added test/maps/legacy/notes.xml   test map

**Comment by insilmaril** [comment]
No response, seems to work.
